### PR TITLE
Plasma statues get fixed

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
tweak: Igniting plasma statues no longer ignores ignition temperature and only creates as much plasma as was used in its creation. 
/:cl:

As a totally unintended side effect plasma statues will no longer automatically create lethal temperatures and will only spawn plasma gas proportional to the amount of plasma used in its creation (approx. 1/8th of the current amount).

### CODE REVIEW

**Comment by ShiggyDiggyDo** [comment]
i ded

**Comment by Buggy123** [comment]
♪♫♪I just want to set the world on fire...♪♫♪

:-1:

**Comment by Robustin** [comment]
To spell it out. You used 5 plasma to create the statue and it would create 40 plasma worth of gas when ignited. 

Now it just checks the amount of ore used to create the statue and spawns gas based on the ratio used for the mineral (10).

**Comment by Iatots** [comment]
Plasma stack stockpile when

**Comment by Armhulen** [comment]
i ded

**Comment by Armhulen** [comment]
""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""fix""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""""

**Comment by Xhuis** [comment]
I think you should wait at least 48 hours before making a PR based on a round's events.

**Comment by CitrusGender** [comment]
why is everyone memeing this post

**Comment by DaxDupont** [comment]
ided

**Comment by BeeSting12** [comment]
@CitrusGender someone set a plasma statue on fire and killed 20 people

**Comment by DaxDupont** [comment]
Also this is a balance change, not a fix.

File: code/game/objects/structures/statues.dm [hunk 1]
```diff
@@ -177,8 +177,8 @@
 	else
 		return ..()
 
-/obj/structure/statue/plasma/proc/PlasmaBurn()
-	atmos_spawn_air("plasma=400;TEMP=1000")
+/obj/structure/statue/plasma/proc/PlasmaBurn(exposed_temperature)
+	atmos_spawn_air("plasma=oreAmount*10;TEMP=[exposed_temperature]")
```
Comment (nicbn) [inline code/game/objects/structures/statues.dm]: eeeeh I think `oreAmount*10` is supposed to be enclosed with brackets

Comment (nicbn) [reply code/game/objects/structures/statues.dm]: Also oreAmount is fixed so why even use it

**Comment by Hatterhat** [comment]
itt: unrobustin gets cucked by one (1) plasma statue

**Comment by ShiggyDiggyDo** [comment]
>giving your own PR a thumbs up

**Comment by Bombpants** [comment]
Leave the plasma mems

**Comment by Robustin** [comment]
I've (ab)used plasma statues more than anyone shitposting here. 

But I also recognize there's no real excuse for letting people vomit a room-destroying item out of their bag instantly, especially when said item is only so powerful because it turns 5 plasma (readily available at roundstart) into 40 worth of gas along with instantly heating the gas to lethal temps. 

This wasn't based on a single round, but I've seen this meme pick up momentum and used at least 5 times this week and so it jumped up about 50 places on my "todo" list.

**Comment by DaxDupont** [comment]
>it wasn't about a round guise
>makes pr 5 minutes after the round ends

**Comment by Robustin** [comment]
I've got dozens of things I want to change but limited time and attention.

I realized there was a problem with statues months ago when I ignited 20 plasma but it made a fire only a fraction of the size of a single (5 plasma) statue. 

Some of these changes are literally something I've wanted to do for years. Like stopping flashes from burning out BEFORE you use them, Kor complained about it in OOC earlier this week, it reminded me of how annoying the issue is, so I finally got around to fixing it.

**Comment by ShiggyDiggyDo** [comment]
>abuse plasma statues
>haha lol 

>die from one being (ab)used
>"But I also recognize there's no real excuse for letting people vomit a room-destroying item out of their bag instantly"

**Comment by Robustin** [comment]
>One
>Implying it wasn't 7-8 being dumped out of a bag multiple times per second

**Comment by XDTM** [comment]
>Overpowered feature
>Someone notices, likely the victim
>Victim makes balance PR on the overpowered feature
>"ided PR, close"
>PR is closed

If something's legitimately op how is someone supposed to fix it? Even if it's robustin, we're talking about 8x gas from plasma sheets, which was likely not intentional from the coder, and definitely not intuitive for players.

**Review by Cyberboss — CHANGES_REQUESTED** [review]
What @nicbn said

File: code/game/objects/structures/statues.dm (hunk 1 above)
Comment (Cyberboss) [reply code/game/objects/structures/statues.dm]: always prefer a variable if it exists

**Comment by DaxDupont** [comment]
Why is it a tweak now, it's still a balance change!

**Comment by optimumtact** [comment]
Maintainer fiat, this is obviously a fucking fix, because it's a consistency issue that it doesn't work properly in the first place.

**Comment by FantasticFwoosh** [comment]
@Iatots [you are correct, he did to nerf plasma](https://github.com/tgstation/tgstation/pull/27761/files/15edeaca91188ee726a6e2f648c79b824f9528f7#r118882798)

**Comment by CPTANT** [comment]
Increase the amount of gas 1 plasma spawns.

**Comment by duncathan** [comment]
Good fix

**Pull request merged**